app/cli/commandlineparser.cpp

```diff
@@ -464,3 +467,46 @@ QString StreamCommandLineParser::getAppName() const
 {
     return m_AppName;
 }
+
+ListCommandLineParser::ListCommandLineParser()
+{
+}
+
+ListCommandLineParser::~ListCommandLineParser()
+{
+}
+
+void ListCommandLineParser::parse(const QStringList &args)
+{
+    CommandLineParser parser;
+    parser.setupCommonOptions();
+    parser.setApplicationDescription(
+        "\n"
+        "List the available apps on the given host as CSV:\n"
+        "\tName, ID, HDR Support, App Collection Game, Hidden, Direct Launch, Path to Boxart"
+    );
+    parser.addPositionalArgument("list", "list available apps");
+    parser.addPositionalArgument("host", "Host computer name, UUID, or IP address", "<host>");
```

Comment:
CSV is nice for consuming in other applications but isn't very friendly for human consumption. I think `list` should just print the app names in a list by default and have a separate `--csv` flag for CSV output.